src/data/reducers/assetsFilters.js

```diff
@@ -0,0 +1,20 @@
+import { FILTER_UPDATED } from '../actions/assets';
+
+const initialState = {
+  courseId: 'course-v1:edX+DemoX+Demo_Course',
```

Comment:
Probably don't want to store this `courseId` as a default forever. This'll serve our purposes for now -- in the future, we'll need some way to get `courseId` from Django-land into Redux-land, so we don't have to worry about hardcoding anything.